packages/server/graphql/mutations/helpers/notifications/MSTeamsNotifier.ts

```diff
@@ -162,6 +162,21 @@ export const MSTeamsNotificationHelper: NotificationIntegrationHelper<MSTeamsNot
     const meetingEndedReviewColumn = new AdaptiveCards.Column()
     meetingEndedReviewColumn.width = 'auto'
 
+    if (summary) {
+      const aiSummaryColumnSet = new AdaptiveCards.ColumnSet()
+      const aiSummaryColumn = new AdaptiveCards.Column()
+      aiSummaryColumn.width = 'stretch'
+      const aiSummaryTitle = new AdaptiveCards.TextBlock('AI Summary 🤖')
+      aiSummaryTitle.weight = AdaptiveCards.TextWeight.Bolder
+      const cleanedAISummary = summary.replace(/(\r\n|\n|\r)/gm, '') // remove line breaks from the summary
```

Comment:
-1 shouldn't line breaks be replaced with spaces?

Reply:
The line breaks happen at the beginning of the summary, so we can just remove them, e.g. `\n\nThe text is saying that something is great and that it was done well.'`

Reply:
There is no guarantee that the bot will reply this way. It can as well write multiple paragraphs where we wouldn't want to replace it with nothing. If it's about leading/trailing whitespace just call `.trim()`

Reply:
Sounds good - updated!